recod_frappe_devtools/build_docs/setup_docs.py

```diff
@@ -0,0 +1,350 @@
+        self.add_sidebars()
+        self.add_breadcrumbs_for_user_pages()
+
+    def add_breadcrumbs_for_user_pages(self):
+        for basepath, folders, files in os.walk(os.path.join(self.docs_path,
+                                                             'user')):  # pylint: disable=unused-variable
+            for fname in files:
+                if fname.endswith('.md') or fname.endswith('.html'):
+                    add_breadcrumbs_tag(os.path.join(basepath, fname))
+
+    def add_sidebars(self):
+        '''Add _sidebar.json in each folder in docs'''
+        for basepath, folders, files in os.walk(self.docs_path):  # pylint: disable=unused-variable
+            with open(os.path.join(basepath, '_sidebar.json'), 'w') as sidebarfile:
+                sidebarfile.write(frappe.as_json([
+                    {"title": "Search Docs ...", "type": "input", "route": "/search_docs"},
+                    {"title": "Docs Home", "route": "/docs"},
+                    {"title": "{} - Docs Home".format(self.app_context['app'].get("title")),
+                     "route": "/docs/{}".format(self.target_app)},
+                    {"title": "{} - User Guide".format(self.app_context['app'].get("title")),
+                     "route": "/docs/{}/user".format(self.target_app)},
+                    {"title": "{} - Server API".format(self.app_context['app'].get("title")),
+                     "route": "/docs/{0}/{1}/api".format(self.target_app, self.app_context["app"].get("docs_version"))},
+                    {"title": "{} - Models (Reference)".format(self.app_context['app'].get("title")),
+                     "route": "/docs/{0}/{1}/models".format(self.target_app,
+                                                            self.app_context["app"].get("docs_version"))},
+                    {"title": "{} - Improve Docs".format(self.app_context['app'].get("title")), "route":
+                        "{0}/tree/{1}/{2}/docs".format(self.docs_config.source_link,
+                                                       self.app_context["app"].get("branch"), self.app)}
+
+                ]))
+
+    def copy_user_assets(self):
+        '''Copy docs/user and docs/assets to the target app'''
+        print('Copying docs/user and docs/assets...')
+        shutil.rmtree(os.path.join(self.docs_path, 'user'),
+                      ignore_errors=True)
+        shutil.rmtree(os.path.join(self.docs_path, 'assets'),
+                      ignore_errors=True)
+        shutil.copytree(os.path.join(self.app_path, 'docs', 'user'),
+                        os.path.join(self.docs_path, 'user'))
+        shutil.copytree(os.path.join(self.app_path, 'docs', 'assets'),
+                        frappe.get_app_path(self.target_app, 'www', 'docs', self.target_app, 'assets'))
+
+        # copy index
+        shutil.copy(os.path.join(self.app_path, 'docs', 'index.md'),
+                    frappe.get_app_path(self.target_app, 'www', 'docs', self.target_app))
```

Comment:
```suggestion

        # copy user guide if exists
        app_user_docs = os.path.join(self.app_path, 'docs', 'user')
        if os.path.exists(app_user_docs):
             shutil.copytree(app_user_docs,
                os.path.join(self.docs_path, 'user'))

        # copy assets if exists
        app_assets_docs = os.path.join(self.app_path, 'docs', 'assets')
        if os.path.exists(app_assets_docs):
            shutil.copytree(app_assets_docs,
                frappe.get_app_path(self.target_app, 'www', 'docs', self.target_app, 'assets'))

        # copy index if exists
        app_index_docs = os.path.join(self.app_path, 'docs', 'index.md')
        if os.path.exists(app_index_docs):
            shutil.copy(app_index_docs,
                frappe.get_app_path(self.target_app, 'www', 'docs', self.target_app))

```

@AminovE99 You did not use the `version-11-hotfix` as the base for this since those lines had an issue already fixed